### ISSUE TITLE
crash

### ISSUE THREAD

**Issue by InitialD09** [NadPlayz/nad_rockstar#2]
why do i crash when i record in an mlo then when i try to add a sequence i crash

**Comment by QBLes** [comment]
> 

This is due to the MLO being escrow (tebex) lets hope they find a way to bypass this problem..

**Comment by Gameadictive** [comment]
my crash is when i try to edit a clip.

**Comment by Wendigo23** [comment]
Yes, if you try to edit a clip while you have a single encrypted/escrowed resource in your server when you made the clip, it will crash.

**Comment by ELOG10** [comment]
Did you find a solution for the crashes?

**Comment by QBLes** [comment]
> Did you find a solution for the crashes?

No this is all up to CFX to fix it..
